autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -682,18 +697,10 @@ def score_with_y_pred_proba(self, y, y_pred_proba):
         else:
             return self.eval_metric(y, y_pred_proba)
 
-    def autotune(self, X_train, X_holdout, y_train, y_holdout, model_base: AbstractModel):
+    # TODO: Refactor
+    def _autotune(self, X_train, X_holdout, y_train, y_holdout, model_base: AbstractModel):
```

Comment:
documentation

Reply:
added additional warnings regarding usage, this is tied to feature_prune functionality which isn't implemented at present.